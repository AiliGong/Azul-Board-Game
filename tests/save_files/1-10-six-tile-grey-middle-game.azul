#myformat
-1
6
1
a
b
BYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULOBYRULO
%6
%6
%6
%6
%6
turn 1 B 2
turn 2 L 3
turn 3 U 3
turn 4 B 2
turn 0 B 2
turn 0 R 5
turn 0 Y 4
turn 0 L 3
turn 0 U 3
turn 5 L 3
turn 0 O 5
turn 0 B 2
turn 0 Y 4
turn 1 R 1
turn 2 U 1
