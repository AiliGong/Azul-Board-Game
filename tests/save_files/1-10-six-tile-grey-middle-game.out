Factories:
0: F U L O B Y R
1:
2:
3: L O B Y
4: R U L O
5: B Y R U

Score for a: 5
Mosaic for a:
1:           R || . . . . . . 
2:         . . || . . . . . B 
3:       . . . || . . . . . U 
4:     . . . . || . . . . . Y 
5:   . . O O O || . . . . . . 
6: . . . . . . || . . . . . . 
broken:

Score for b: 3
Mosaic for b:
1:           U || . . . . . . 
2:         . . || . . . . . B 
3:       . . . || . . . . . L 
4:     . . . . || . . . . . . 
5:   . . R R R || . . . . . . 
6: . . . . . . || . . . . . . 
broken:
